src/lib/actions/maintenance.ts

```
@@ -54,6 +59,65 @@ function notRunningBackupSkipMessage(name: string): string {
+    if (/^Agent '[^']+' not found: .+\/manifest\.yaml$/.test(message)) {
+      orphanManifestMessage = message;
+    } else {
+      backupError = err;
+      hasBackupError = true;
+    }
+  } finally {
+    if (!relockBackupShieldsWindow(sandboxName, window, true, shieldsWindowOptions)) {
+      throw new Error(
+        `Shields lockdown could not be restored for '${sandboxName}' after backup-all; aborting remaining backups.`,
+        backupError instanceof Error ? { cause: backupError } : undefined,
+      );
```

Biome (2.5.3)

[error] Unsafe usage of 'throw'.

(lint/correctness/noUnsafeFinally)